doc/design/error-handling.md

```diff
@@ -130,13 +125,12 @@ If the error cannot be handled at a higher level by the caller, the error handle
 
 Examples include wrapping third party API functions that return error codes or obtaining a value from a container when this can fail for some reason (e.g. container is empty). If no additional information about the error is available or required, ``cxx::optional<T>`` can be used instead.
 
-
-## Error Handling in posh
+#### Error Handling in posh
```

Comment:
I think this a subchapter of "Error Handling" and not of "Expects and Ensures". If I'm right, please remove one ``#``. Same for "Error Handling in utils".

Reply:
Nice catch, thanks! Fixed